blockbase/theme.json

```diff
@@ -160,8 +160,8 @@
 			},
 			"post-content": {
 				"padding": {
-					"left": "var(--wp--custom--margin--horizontal)",
-					"right": "var(--wp--custom--margin--horizontal)"
+					"left": "20px",
+					"right": "20px"
```

Comment:
After a 3rd look I'm unsure why this value isn't `var(--wp--custom--post-content--padding--left)` and `var(--wp--custom--post-content--padding--right)`.  If the post-content value needs to change from 30px to 20px then we should change it in the custom block so that the other places that leverage that value are picking up what's expected.

Reply:
I think you mean these custom padding variables should be set to `var(--wp--custom--margin--horizontal)` and we should change the value of that custom variable to `20px`. Or am I misunderstanding?

Reply:
I'm saying that the "post-content" padding style should be set to the custom variable `--wp--custom--post-content--padding--left/right`.  I don't believe it was ever correct to be set to `--wp--custom--margin--horizontal`, but because the custom value was also set to the same variable it was never perceived as an issue.

That variable (`--wp--custom--post-content--padding--left/right`) is currently being set to `--wp--custom--margin--horizontal` but if we want to reduce the padding around post-content we should be adjust that custom value (which other things are using) instead of changing the value here.

I'm not sure why that value should be 20px instead of the 30px that --wp--custom--margin--horizontal is, but I'm saying that it's the custom value that should change (so that other things in CSS know about the change).

TL;DR
```
custom: {
  post-content: {
    padding: {
      left: "20px"
      right: "20px"
    }
  }
}
...
styles: {
    post-content: {
      padding: {
        left: var(--wp--custom--post-content--padding--left)
        right: var(--wp--custom--post-content--padding--left)
```
so that elsewhere...
```
    margin-left: calc(-1 * --wp--custom--post-content--padding--left);
```

Reply:
Let be known that I'm completely confused and it's right and I'm not sure why I'm arguing.... :P